### PR TITLE
Startup #99

### PR DESCRIPTION
**Pull request by SPlanzer**
Resolves CI `ImportError: cannot import name 'Config'` error

### CODE REVIEW

**Comment by SPlanzer** [comment]
@strk 
the docker image is failing. to get this to work I am using FROM `qgis/qgis:release-3_10`

however against #96 you said 
> Addressing a QGIS branch rather than QGIS tag (makes builds less predictable)

I am getting this image from https://hub.docker.com/r/qgis/qgis/tags. Which image should I be using considering  `qgis/qgis:final-3_10_4_disco` is failing (see #92)?

**Comment by strk** [comment]
> I am getting this image from https://hub.docker.com/r/qgis/qgis/tags. Which image should I be using considering `qgis/qgis:final-3_10_4_disco` is failing (see #92)?

Could you try `final-3_10_5` ?

**Comment by SPlanzer** [comment]
> Could you try final-3_10_5 ?

Yip, updated to final-3_10_5, push and passed CI

File: .docker/qgis/scripts/enable_nzgbplugin [hunk 1]
```diff
@@ -2,6 +2,9 @@
 
 CONFIGFILE=/root/.local/share/QGIS/QGIS3/profiles/default/QGIS/QGIS3.ini
 
+# Install startup.py
+cp /scripts/startup.py /root/.local/share/QGIS/QGIS3/startup.py
```
Comment (strk) [inline .docker/qgis/scripts/enable_nzgbplugin:6]: Did you forget to `git add` the `startup.py` script ? I don't see it in teh diff screen

Comment (SPlanzer) [reply .docker/qgis/scripts/enable_nzgbplugin:6]: Yip. Added

**Comment by SPlanzer** [comment]
> The fact that CI was green even without that file suggests it would be useful to have CI also run the make docker-up target (and then we'd want that target to fail if one of the containers do not stay on) - but can be postponed if you don't have time now

Next CI step is to integrate pyhton integration tests. This will indicate any issues with containers

**Pull request merged**